hackathon_site/hardware/test_api.py

```diff
@@ -508,7 +535,7 @@ def test_user_has_view_permissions(self):
         self.assertEqual(expected_response, data["results"])
 
     def test_hardware_get_success(self):
-        self._login(self.permissions)
+        self._login(self.all_hardware_permissions)
```

Comment:
It would be better to keep all of these scoped to just the view permissions that they originally were